README.md

```diff
@@ -31,13 +31,13 @@ input CreateThing {
 
 ## Build
 
-This project uses [.NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/) and requires .NET 7.0+ runtime. In order to build the project locally run
+This project uses [.NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/) and requires .NET 7.0+ runtime. In order to build the project locally, run:
 
 ```shell
 dotnet build
```

Comment:
Should this be run in the root of the project or in `Server`? We make the distinction down below when we _run_ the server, but not when building it. It seemed to work from the root, but it would be great to specify.